src/overlays/actors/ovl_Bg_Toki_Hikari/z_bg_toki_hikari.c

```diff
@@ -130,8 +128,9 @@ void func_808BA274(BgTokiHikari* this, PlayState* play) {
     }
 }
 
-void func_808BA2CC(BgTokiHikari* this, PlayState* play) {
-    s32 pad[2];
+void func_808BA2CC(Actor* thisx, PlayState* play) {
+    BgTokiHikari* this = (BgTokiHikari*)thisx;
+    PlayState* play2 = (PlayState*)play;
```

Comment:
is play2 out of habit or required

Reply:
uh habit. Don't think it matters much so I'll remove it